website/docs/spec/index.md

```diff
@@ -179,6 +183,30 @@ The message encoding and schema must match that of the Channel record correspond
 | 8     | publish_time | Timestamp | Time at which the message was published. If not available, must be set to the log time.                         |
 | N     | data         | Bytes     | Message data, to be decoded according to the schema of the channel.                                             |
 
+### Secondary Index Key (op=0x10)
```

Comment:
I think we should come up with a more future-proof word than "secondary", in case we decide in a V2 to merge the primary indexes into this same record type.